.github/workflows/error-ref-publisher.yaml

```
@@ -0,0 +1,63 @@
+name: Error Code Reference Publisher
+on:
+  push:
+    branches:
+      - "master"
```

```suggestion
      - 'master'
```
My OCD says it would be cool to preserve those single quotes.

---

Hey @DelusionalOptimist I also faced this problem like if we are saving this file in either windows or mac the single quote changes to double quote automatically but it doesn't happen in windows. So any suggestion how to solve this.

---

Not sure. I guess this depends on your text editors or extensions.
But we've been sticking with single quotes in rest of the workflows so I'd say let's go with the same here as well so as to not accidentally break stuff XD.

---

Okay lets check how to solve this.